docs/sources/mimir/release-notes/v2.9.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Grafana Mimir version 2.9 release notes"
+menuTitle: "V2.9 release notes"
+description: "Release notes for Grafana Mimir version 2.9"
+weight: 300
+---
+
+# Grafana Mimir version 2.9 release notes
+
+Grafana Labs is excited to announce version 2.9 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Experimental support for optimized series selection strategy** Reduce the volume of data necessary to complete requests with individual label selectors that are present on a lot of series.
+- **Reduced CPU utilisation for some shuffle sharding scenarios** Mimir queriers will now use significantly less CPU in cases where shuffle sharding is enabled with a high number of shards.
+- **Reduced object storage API calls in compactors and rulers** Mimir 2.9 comes with optimizations that will reduce the amount of times compactors and rulers need to access rules stored in object storage.
+- **Experimental support for streaming chunks from ingester to querier** This is expected to greatly reduce querier memory consumption when evaluating queries that select a large number of series, because chunks streamed from the querier can now be read into memory as needed.
```

Comment:
@charleskorn Do you want to add more color here?

Reply:
I don't have any more specifics to add just yet - once we've done some more testing we might have something more to add. 

I'm OK for this to go as-is, we can talk about it more in the next release (and the next release will also include a bunch of improvements that didn't make this release).